### ISSUE TITLE
Attachments not send when < 3 MB

### ISSUE THREAD

**Issue by flokle** [nylas/nylas-java#222]
SDK 2.2.1

In Messages.kt:
```
 return if (attachmentSize >= FileUtils.MAXIMUM_JSON_ATTACHMENT_SIZE) {
      val attachmentLessPayload = requestBody.copy(attachments = null)
      val serializedRequestBody = adapter.toJson(attachmentLessPayload)
      val multipart = FileUtils.buildFormRequest(requestBody, serializedRequestBody)

      client.executeFormRequest(path, NylasClient.HttpMethod.POST, multipart, responseType)
    } else {
      val serializedRequestBody = adapter.toJson(requestBody)
      createResource(path, serializedRequestBody)
    }
```

As the `attachments` field on SendMessageRequest is marked as transient, this part of the conditional statement
```
val attachmentLessPayload = requestBody.copy(attachments = null)
```
is useless 
and the else part ignores attachments.

**Comment by atejada** [comment]
Thanks @flokle we're going to take a look, fix it and let you know 😉

**Comment by mrashed-dev** [comment]
Hey @flokle just wanted to let you know that this is fixed and is currently released as part of v2.3.0!
